rep-0155.rst

```diff
@@ -0,0 +1,902 @@
+requirements of the application, the available sensing capabilities, and the
+position/behaviour of the humans, only some might be available for a given
+person, at a given time.
+
+Person Identifier
+-----------------
+
+The **person identifier** MUST be a unique ID (typically, a UUID) permanently
+associated with a unique person. This person ID is normally assigned by a node
+able to perform person identification (face recognition node, voice
+recognition node, sound source localisation + name, identification based on
+physical features like height/age/gender, person identification based on
+pre-defined features like the colour of the clothes, etc.) This ID is meant to
+be **persistent** so that the robot can recognise people across
+encounters/sessions. Nodes providing person IDs MAY serialise these IDs to a
+permanent storage, for them to persist across robot reboots.
+
+When meaningful (see section `Person frame`_), a TF frame MUST be
+associated to the person ID and named ``person_<personID>``. Due to the
+importance of the head in human-robot interaction, the ``person_<personID>``
+frame is expected to be placed as close as possible to the head of the human.
+If neither the face nor the skeleton are tracked, the ``person_<personID>``
+frame might be located to the last known position of the human, or removed
+altogether if no meaningful estimate of the human location is available. See
+section `Person frame`_ for details regarding the ``person_<personID>`` frame.
+
+At any given time, the list of known persons is published under the
+``/humans/persons/tracked`` topic as ``hri_msgs/IdsList`` messages.
+
+Merging Person Identifiers
+''''''''''''''''''''''''''
+
+In certain cases, two person IDs must be merged (for instance, the robot
+detects that a voice and a face that were thought to belong to different people
+are indeed the same person).
+
+In such a case, one of the person ID is marked as an *alias* of the other
+person, by publishing the ID of the other person on a special subtopic named
+``alias``. See section `Topics structure`_ for details.
+
+.. Note:: The reverse operation (spliting a person into two) can be realised
+   by simply publishing a second person ID.
+
+Anonymous persons
+'''''''''''''''''
+
+While person IDs are generally expected to be permanent, one exception exists
+for persons that the robot has detected but not yet identified.
+
+For instance, the robot hears a voice, and therefore knows that a person is
+around, but no voice identification nodes is available -- or the voice
+identification has not yet recognised the voice: in such a case, an *anonymous
+person* might be created, ie a person who has not yet been assigned a permanent
+ID.
+
+*Anonymous persons* are treated like regular persons. They however publish a
+latched ``true`` boolean on their ``/anonymous`` subtopic, and their ID is not
+guaranteed to be permanent (it can in fact change/be removed at any point).
```

Comment:
what other messages do you have in mind? (the rationale for using a subtopic here was to ensure that any node dealing with persons would transparently see 'anonymous' persons  as well. But I guess this could be modeled differently...)